react/components/EXPERIMENTAL_FilterBar/UncoupledStatement.js

```diff
@@ -0,0 +1,144 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import SubjectAtom from '../EXPERIMENTAL_Conditions/Atoms/SubjectAtom'
+import VerbAtom from '../EXPERIMENTAL_Conditions/Atoms/VerbAtom'
+import ObjectAtom from '../EXPERIMENTAL_Conditions/Atoms/ObjectAtom'
+
+class Statement extends React.Component {
+  handleChangeStatement = (newValue, structure) => {
+    this.props.onChangeStatement(newValue, structure)
+  }
+
+  resetPredicate = subjectValue => {
+    const { options } = this.props
+
+    this.handleChangeStatement(
+      options[subjectValue].verbs[0].value ||
+        Statement.defaultProps.statements[0].verb,
+      'verb'
+    )
+
+    this.handleChangeStatement(
+      Statement.defaultProps.statements[0].object,
+      'object'
+    )
+
+    this.handleChangeStatement(null, 'error')
+  }
+
+  render() {
+    const {
+      options,
+      subjectPlaceholder,
+      isFullWidth,
+      statements,
+      statementIndex,
+      omitSubject,
+      omitVerbs,
+      onChangeObjectCallback,
+    } = this.props
+    const condition = statements[statementIndex]
+    const atomProps = {
+      statements: statements,
+      options: options,
+      isFullWidth: isFullWidth,
+      statementIndex: statementIndex,
+      onChangeObjectCallback: onChangeObjectCallback,
+    }
+
+    return (
+      <div>
+        <div className="flex-column w-100 mv3">
+          <div
+            className={`flex w-100 items-start ${
+              isFullWidth ? 'flex-column items-stretch' : ''
+            }`}>
+            {omitSubject
+              ? omitVerbs
+                ? [<ObjectAtom key="object" {...atomProps} />]
```

Comment:
This doesn't need to be an array

Reply:
just to give context, before there was a prop `isRTL` that it runned a .reverse() in this array to adapt to eastern languages where speech structure is Object, Verb, Subject instead of Subject, Verb, Object.

Since I removed it, the array is not needed any more. Forgot it there.

Reply:
Oh, nice. Ok 👌